editions/tw5.com/tiddlers/widgets/ButtonWidget.tid

```diff
@@ -9,11 +9,11 @@ type: text/vnd.tiddlywiki
 
 The button widget displays an HTML `<button>` element that can perform a combination of optional actions when clicked:
 
-* Executing any ActionWidgets passed in the `actions` attribute
-* Executing any ActionWidgets that are immediate children of the button widget
+* Executing any ActionWidgets passed in the `actions` attribute (''best practice'')
```

Comment:
I think it would be clearer to say `Executing any ActionWidgets passed in the actions attribute, which is the preferred approach`.